4_arm.ipynb

```diff
@@ -62,13 +115,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "各関節に対応するリンクがロボットのどの位置にあるか確認しましょう。\n",
     "\n",
     "rvizを立ち上げて、「TF」を展開して「Frames」から興味のあるリンク名にチェックを入れてその位置を確認しましょう。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's check the name joints on the robot.\n",
+    "\n",
+    "Launch RViz, expand \"TF\", check the link name you are interested in from \"Frames\" item, and see its location."
```

Comment:
"Launch RViz, expand \"TF\", check the link name you are interested in from \"Frames\" item, and check its location."